addons/website/controllers/main.py

```diff
@@ -661,6 +661,12 @@ def google_maps_api_key(self):
             'google_maps_api_key': request.website.google_maps_api_key or ''
         })
 
+    @http.route('/website/is_image_optimization_disabled', type='json', auth='user', website=True)
+    def image_optimization_disabled(self):
+        if request.env['ir.config_parameter'].sudo().get_param('website.no_quality_optimization'):
+            return True
+        return False
```

Comment:
Doesn't seems right, this does not consider the global ICP which should also disable it in the frontend?